pymatgen/io/lobster/inputs.py

```diff
@@ -1,6 +1,6 @@
-"""
-Module for reading Lobster input files. For more information
-on LOBSTER see www.cohp.de.
+"""Module for reading Lobster input files.
```

Comment:
```suggestion
"""Module for reading LOBSTER input files.
```

Reply:
Oops... I tried to correct cases for `LOBSTER` (which refers to the package, instead of the class), and it seems I missed some. Thanks for catching this!